benchmark/scenarios/_request_load.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Request load scenario for the control service benchmarks.
+"""
+from itertools import repeat
+
+from zope.interface import implementer
+from eliot import start_action, write_failure, Message
+from eliot.twisted import DeferredContext
+
+from twisted.internet.defer import CancelledError, Deferred
+from twisted.internet.task import LoopingCall
+
+from flocker.common import loop_until, timeout
+
+from .._interfaces import IScenario
+
+from ._rate_measurer import RateMeasurer, DEFAULT_SAMPLE_SIZE
+
+
+class RequestRateTooLow(Exception):
+    """
+    The request rate dropped below a threshold.
+    """
+
+
+class RequestRateNotReached(Exception):
+    """
+    The request rate did not reach the target level.
+    """
+
+
+class RequestOverload(Exception):
+    """
+    There are too many outstanding requests.
+    """
+
+
+class NoNodesFound(Exception):
+    """
+    No nodes were provided by the control service.
+    """
+
+
+@implementer(IScenario)
+class RequestLoadScenario(object):
+    """
+    A scenario that places load on the cluster by performing
+    requests at a specified rate.
+
+    :ivar reactor: Reactor to use.
+    :ivar scenario_setup: instance of an implementation of the interface
+        `IRequestScenarioSetup`.
```

Comment:
Also, use double backtick for references like this, eg \`\`IRequestScenarioSetup\`\`